webcrypto/subtle_crypto_test.go

```diff
@@ -70,13 +70,13 @@ func TestSubtleCryptoGenerateKey(t *testing.T) {
 func TestSubtleCryptoImportExportKey(t *testing.T) {
 	t.Parallel()
 
-	t.Run("symmetric", func(t *testing.T) {
+	t.Run("importKeys", func(t *testing.T) {
```

Comment:
Is there a reason for this sub test ?

Seems like either the arguments to `executeTestScripts` should be in two subtest or this sutest should just go.

Reply:
yep, you're right will split them